website/templates/accessor/sensor.html.mako

```diff
@@ -58,21 +58,7 @@
 
 <%block name="elem_links_row">
 % if show_add_sensor:
-    <form id="add_sensor_form" action="${url_for('accessor.add_sensor')}" method="post" class="inline-form">
-        <div class="input-group">
-            <div class="input-group-prepend">
-                <span class="input-group-text">New sensor</span>
-            </div>
-            <input id="sensor_name" type="text" name="name" class="form-control" placeholder="Sensor name" maxlength="64" required/>
-            <input id="sensor_secret" type="text" name="secret" class="form-control" placeholder="Sensor secret" maxlength="64" required/>
-            <div class="input-group-append">
-                <button class="btn btn-outline-secondary" type="submit">
-                    <i class="fas fa-plus-circle"></i>
-                </button>
-            </div>
-        </div>
-    </form>
-    <br>
+    <%include file="../add_sensor.html.mako" />
```

Comment:
I don't think we should go too much the way of having parts of html here and there.

If we really want something common for both pages (which I'm not sure we do since in the admin page we probably want to be able to select the owner without having to then edit it) then we should create a mako function in `generic.html.mako`

Reply:
Ok, a mako function sounds good, wich doesn't prevent us from also having a drop down for the owner on the sensor page too, keeping the 2 the same.

Reply:
I reverted all that and stashed it so when this MR is done we can reintroduce it in a slightly cleaner way (i.e. have different behaviours if we're admin so the sensor can be attached to any user).